.github/workflows/test.yml

```diff
@@ -45,6 +45,8 @@ jobs:
           poetry-${{ hashFiles('**/poetry.lock') }}
     - name: set poetry config path
       run: poetry config virtualenvs.path ~/.virtualenvs
+    - name: update pip 
```

Comment:
We don't _need_ this step. Let's take it out.

Reply:
The pipeline was breaking before I added it. I'll remove it now though to ensure it matches devops spec.